scenarioo-client/test/protractorE2E/specs/manage_branch_aliases.ts

```diff
@@ -1,7 +1,6 @@
 'use strict';
 import {scenario, step, useCase} from "scenarioo-js";
 
-var scenarioo = require('scenarioo-js');
 var pages = require('./../webPages');
```

Comment:
@danielsuter did you open an issue for getting rid of the requires in all tests? also the unused scenarioo is somehow in all tests --> open an issue for we not forget to clean this up. Thanks a lot!

Reply:
@bruderol  Thanks for the reminder, I opened #652